src/internal/git/utils.go

```diff
@@ -43,33 +48,33 @@ func MutateGitUrlsInText(host string, text string, gitUser string) string {
 }
 
 func transformURLtoRepoName(url string) (string, error) {
-	// For further explanation: https://regex101.com/library/UfILls and https://regex101.com/rary/UfILls
-	findRegex := regexp.MustCompile(`\/([\w\-]+)(.git)?(@([\w\-\.]+))?$`)
-	substrings := findRegex.FindStringSubmatch(url)
-	if len(substrings) == 0 {
+	matches := gitURLRegex.FindStringSubmatch(url)
+	if len(matches) != 6 {
 		// the first element in the return substrings is
```

Comment:
I must of borked up this comment when my PR got merged. Could you fix it up in your PR?